packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/normal-dialing.js

```diff
@@ -134,4 +193,266 @@ describe('Call With Screenshare', () => {
+      expect(browserChrome.$('#register').isEnabled()).to.be.false;
+      expect(browserChrome.$('#unregister').isEnabled()).to.be.false;
+      expect(browserChrome.$('#dial').isEnabled()).to.be.false;
+      expect(browserChrome.$('#invitee').isEnabled()).to.be.false;
+      expect(browserChrome.$('#hangup').isEnabled()).to.be.false;
+      expect(browserChrome.$('#share-screen').isEnabled()).to.be.false;
+      expect(browserChrome.$('#stop-screen-share').isEnabled()).to.be.false;
+
+      expect(browserFirefox.$('#login').isEnabled()).to.be.true;
+      expect(browserFirefox.$('#logout').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#register').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#unregister').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#dial').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#invitee').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#hangup').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#share-screen').isEnabled()).to.be.false;
+      expect(browserFirefox.$('#stop-screen-share').isEnabled()).to.be.false;
+    });
+
+    it('click login button for both browsers', () => {
+      browserChrome.$('#login').click();
+      browserFirefox.$('#login').click();
+    });
+
+    it('enters user login information for both browsers', () => {
+      const spockEmailInput = browserFirefox.$('#IDToken1');
+      const spockEmailButton = browserFirefox.$('#IDButton2');
+      const spockPasswordInput = browserFirefox.$('#IDToken2');
+      const spockPasswordButton = browserFirefox.$('#Button1');
+
+      const mccoyEmailInput = browserChrome.$('#IDToken1');
+      const mccoyEmailButton = browserChrome.$('#IDButton2');
+      const mccoyPasswordInput = browserChrome.$('#IDToken2');
+      const mccoyPasswordButton = browserChrome.$('#Button1');
+
+
+      // Wait upto 2s to redirect to idbroker and wait for form to render
+      spockEmailInput.waitForExist({timeout: 2000});
+      mccoyEmailInput.waitForExist({timeout: 2000});
+
+      // the login pages aren't known for their consistency, so we'll break with
+      // out pattern of using accessibility labels for test selectors in favor
+      // of using the well known (if not a11y helpful) ids.
+      spockEmailInput.setValue(spock.email);
+      spockEmailButton.click();
+
+      mccoyEmailInput.setValue(mccoy.email);
+      mccoyEmailButton.click();
+
+      spockPasswordInput.waitForExist();
+      spockPasswordInput.setValue(spock.password);
+      spockPasswordButton.click();
+
+      mccoyPasswordInput.waitForExist();
+      mccoyPasswordInput.setValue(mccoy.password);
+      mccoyPasswordButton.click();
+    });
+
+    it('accepts the permissions grant if it appears', () => {
+      const acceptButton = $('button[name="accept"]');
+
+      // Wait upto 2s for grant page to render
+      // Optionally handle the grant page, but don't fail if this is an app that
```

Comment:
This doesn't seem to be optional, it will fail if the grant page appears. But since we are always expecting the grant page for this application, let's just remove the comment about it being optional.